locustempus/main/views.py

```diff
@@ -87,6 +87,13 @@ class CourseDetailView(LoggedInCourseMixin, DetailView):
     model = Course
     template_name = 'main/course_detail.html'
 
+    def get_context_data(self, **kwargs):
+        ctx = super().get_context_data(**kwargs)
+        course: Course = kwargs.get('object')
```

Comment:
Is this supposed to be `course = kwargs.get('object')`? I'm not sure if this is some new python syntax I don't know about...

Reply:
It's a type annotation. I'm trying out MyPy for this project. I forget the version, but as of Python 3.something it's valid Python. The interpreter ignores it at runtime.

Reply:
ah okay, cool!